### ISSUE TITLE
Add a button (with e.g. a globe or keyboard icon) for single-tap switching to the "next" IME

### ISSUE THREAD

**Issue by Kaljurand** [AnySoftKeyboard/AnySoftKeyboard#928]
Allow the user to switch to the next IME (in a sequence of two or more IMEs) by just tapping once.

It is unlikely that a single IME excels at all aspects of (multilingual) text entry, e.g. it might be great for swiping Livonian, but suck at voice typing Võro. Thus an IME should allow the (multilingual) user to quickly switch to another IME (and the other IME should similarly allow the user to switch back to the initial IME).

Newer versions of Android (v5+) make IME switching simpler than before. One needs to tap the (IME-independent) keyboard icon in the lower (or upper, in case of landscape) right corner and then tap a radio button in the "Change keyboard" dialog that pops up. Even quicker (and with much less cognitive load) is to tap a globe/keyboard button integrated into the active IME. This button should call switchToNextInputMethod with "false" on the 2nd argument.

See also: https://developer.android.com/guide/topics/text/creating-input-method.html#Switching

**Comment by chpznf** [comment]
I strongly agree that it should be implemented. It is a must for people who also type in languages that are not aviable yet and need to switch between them fast. I like AnySoftKeyboard a lot but unfortunately it is not usable to me because of this issue as of now.

**Comment by Kaljurand** [comment]
Solving this issue has become more important because other mainstream keyboards still do not have it (e.g. SwiftKey), or have recently dropped it (Gboard).

**Comment by maxrempel** [comment]
Strangely, I do have the globe language switch button on my older phone, but I can not find a way to get it on my new phone. 
The old one has Android 7.0 "Nougat" and the new one has Android 11. 
I thoroughly copied the settings from one to another but this didn't help. 
I wonder, where is this setting - to get the language swticher on the keyboard. 
The space swipe works very poorly - it often types a space or a period instead of switching the language.
